lib/src/model/game/material_diff.dart

```diff
@@ -35,6 +36,29 @@ class MaterialDiff with _$MaterialDiff {
     final IMap<Role, int> blackCount = board.materialCount(Side.black);
     final IMap<Role, int> whiteCount = board.materialCount(Side.white);
 
+    final IMap<Role, int> blackStartingCount =
+        Board.standard.materialCount(Side.black);
+    final IMap<Role, int> whiteStartingCount =
+        Board.standard.materialCount(Side.white);
+    // TODO: parameterise starting position maybe so it can be passed in
```

Comment:
Yes I think this is needed before merging - It's already possible to view (but not play) variants that have a different starting position (e.h. Horde). With this PR, `capturedPieces` will display incorrect values currently.

However, I noticed that the web UI does not display material difference at all for Horde, maybe we should do the same? @veloce wdyt?